include/zephyr/kernel.h

```diff
@@ -6224,6 +6224,17 @@ int k_thread_runtime_stats_enable(k_tid_t thread);
  */
 int k_thread_runtime_stats_disable(k_tid_t thread);
 
+/**
+ * @brief Check if runtime statistics gathering is enabled for a thread
+ *
+ * This function checks the status of runtime statistics gathering for the
+ * specified thread and returns the current usage_lock status.
```

Comment:
This makes no sense.

Reply:
Good catch

Reply:
Do I need to remove it?

Reply:
I think the wording just needs to be changed.

Reply:
How to change to `Checks whether runtime statistics collection is enabled for the specified thread`?

Reply:
@rruuaanng - I like the proprosed change in your comment--it is very clear.